laelf.adoc

```diff
@@ -815,6 +815,38 @@ See <<code_models>> for how it works on various code models.
 |R_LARCH_TLS_DESC_PCREL20_S2
 | 22-bit PC-relative offset to TLS DESC GOT entry
 |`+(*(uint32_t *) PC) [24 ... 5] = (GOT+GD) [21 ... 2]+`
+
+|127
+|R_LARCH_CALL32
```

Comment:
The relocation type `R_LARCH_CALL36` uses 36 effective bits for operand encoding, corresponding to a signed 38-bit PC-relative offset. From the perspective of semantic consistency, wouldn't it make more sense to name `R_LARCH_CALL32` as `R_LARCH_CALL30` instead?

Reply:
I guess so.  @cloudspurs: if you agree I'll change it.

Reply:
I also think so, please change it, thanks.